web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -688,6 +688,19 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                 })}
               </Select>
             </Form.Item>
+            <Form.Item
+              label={formatMessage({ id: 'page.upstream.step.scheme' })}
+              name="scheme"
+            >
+              <Select disabled={readonly}>
+              {['http', 'https', 'grpc', 'grpcs'].map((item, index) => (
```

Comment:
I have never done `.tsx` before.  I have done one other `.tsx` review.  So similar my view is that an array or list should be ordered somehow and mostly its done alphabetically.  I always like rules, standards and style guides. 

```suggestion
              {['grpc', 'grpcs', 'http', 'https'].map((item, index) => (
```